From 2a32fb2e72573c80513e65fc0c8fb071575601d8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Wed, 11 Sep 2019 14:23:44 +0200
Subject: [PATCH 073/152] extensionSystem: Fix a wrong error message

The extension object is added to the `this._extensions` Map inside
`createExtensionObject`, not inside `loadExtension`.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/96
---
 js/ui/extensionSystem.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/extensionSystem.js b/js/ui/extensionSystem.js
index a3b8d615d..47750bda3 100644
--- a/js/ui/extensionSystem.js
+++ b/js/ui/extensionSystem.js
@@ -306,7 +306,7 @@ var ExtensionManager = class {
         let dir = extension.dir;
 
         if (!extension)
-            throw new Error("Extension was not properly created. Call loadExtension first");
+            throw new Error("Extension was not properly created. Call createExtensionObject first");
 
         let extensionJs = dir.get_child('extension.js');
         if (!extensionJs.query_exists(null)) {
-- 
2.23.0

